youtube_dl/extractor/pbs.py

```diff
@@ -685,6 +686,17 @@ def extract_redirect_urls(info):
                         ttml_caption_suffix, '/%d_Encoded.vtt' % (ttml_caption_id + 2)),
                     'ext': 'vtt',
                 }])
+        else:
+            captions = try_get(info, lambda x: x['cc'], dict) or {}
+
+            if captions:
+                subtitles['en'] = []
+                for caption_url in captions.values():
+                    subtitles['en'].extend([{
+                        'ext': re.search(r'\.(\w{3,4})$',
+                                         caption_url).group(1),
+                        'url': caption_url
+                    }])
```

Comment:
reduce indentation by using setdefault method, use `append` instead of `extend` when you're adding a single element at a time:
```suggestion
            for caption_url in (info.get('cc') or {}).values():
                subtitles.setdefault('en', []).append({
                    'url': caption_url
                })
```